packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -92,6 +93,9 @@ export const filters: Record<string,any> = {
   "unnominated2019": {
     nominationCount2019: 0
   },
+  "unnominated": {
+    reviewVoteCount: 0
+  },
```

Comment:
In other reason not to filter allposts with this, I'm not confident there's an index supporting this. `TODO`

Reply:
We can just fix that tho